src/AriesSDK.js

```diff
@@ -105,6 +105,8 @@ const chatCreateCase = (options) => publishMessageToBusAndWaitForResponseWithMat
 
 const retrieveSmsOtp = () => publishMessageToBusAndWaitForResponseWithMatchingId('sms.retrieve-otp');
 
+const openWithDedicatedApp(options) => publishMessageToBusAndWaitForResponseWithMatchingId('view.open-with-dedicated-app', options)
```

Comment:
```suggestion
const openWithDedicatedApp = (options) => publishMessageToBusAndWaitForResponseWithMatchingId('view.open-with-dedicated-app', options);
```

This will fix the broken pipeline as well

Reply:
Good catch!